### ISSUE TITLE
Order Servers for Kamloops Prod

### ISSUE THREAD

**Issue by stewartshea** [BCDevOps/OpenShift4-RollOut#34]
This is broken into multiple orders and delivery steps:
Step 1 (provide infra for Release 1):
- [x] Order VMs for Masters, Infra, Boot Service, and NetApp Virtual OnTap hosts
- [x] Identify workaround for internal storage (**DXC**)

Step 2 (order hardware to replace Virtual and SAN hardware to be ready for build-out in Release 2)
- [x] Order 15 x physical hosts (see spreadsheet for host details) (iStore #1007227,1009749, 1009751, 1009754)
- [x] Confirm physical host order and expected delivery date (**BCGov/DXC**)
- [x] Order additional SSDs for 8 x re-provisioned App physical hosts (**blocked - DXC**)
*Expected delivery June 8, 2020*

Step 3 (Decision to use physical NetApp devices for production clusters has been made)
- [x] Order 2 physical NetApp appliances for production clusters in Kamloops and Calgary
- [x] Confirm order completed and update expected delivery date (**BCGov/DXC**)

**Comment by lukegonis** [comment]
Order drafted, sharing with SB to make sure we have all the correct information, will submit after review.

**Comment by StevenBarre** [comment]
@Joe-Craig has forwarded the info over to our Finance/Procurement team to get the ball rolling on our end while this works though the gov side of iStore.

**Comment by jefkel** [comment]
VMs for Step 1 have been ordered and are only waiting for provision-able networks for that checkbox to be complete.

Holding on order for temporary SAN storage for existing physicals until we have a better idea of when we can actually build the cluster.

Holding on order for physical NetApp until go/no-go decision early Jan, but expect to have SCO ready to submit by the beginning of Jan.

**Comment by jefkel** [comment]
Confirmation that the order for 15 hosts has been submitted and confirmed in iStore (1007227)

**Comment by StevenBarre** [comment]
Sigh, the iStore that came through on our end was only for 1 server. @lukegonis will need to re-submit for the other 14.

**Comment by jefkel** [comment]
@lukegonis has re-submitted the iStore order and brought in the same people that helped with the VM orders to assist.

**Comment by jefkel** [comment]
Confirmation from serviceDesk of 3 more iStore #'s to track:
1009749
1009751
1009754

**Comment by jefkel** [comment]
Luke has updated the iStore orders with the required paperwork, and they should be showing up on the DXC side soon.

**Comment by jefkel** [comment]
@sbarre-esit is chasing the additional storage order for the existing 8 hosts.  The iStore order is blocked as the iStore team is unable to attach the order to the appropriate assets

**Comment by jefkel** [comment]
Looks like SSDs for existing 8 hosts will be included in the 15 physical host purchase, pending confirmation from DXC when SCO is finalized.

**Comment by jefkel** [comment]
@Joe-Craig I've added you in the hopes that you can provide some clarity with regard to the ordering process and any additional items we need to track.

Specifically tasks and status updates for the following:
Order of 15 additional hosts (see above for iStore orders) - iStore submitted, what comes next? what should we track?)
Order of internal SSDs for 8 existing physical hosts - Cannot submit iStore for this, what is our "go forward" plan?
Order of NetApp Physical Appliances for Prod clusters - updates?

**Comment by Joe-Craig** [comment]
@jefkel  I will try and elaborate on where we are with these orders...

- 15 additional hosts: As these are not standard specification machines they have to be ordered on an SCO not just through iStore. We are getting quotes from our vendor on these machines and when they come in our finance will put together a pricing. This is then put into an SCO (Service Offering) which goes to OCIO to be approved. Once the SCO is approved it is now an offering taht can be ordered. So the iStore requests will be updated to request this SCO. Once the iStore is then signed-off we can procure the servers. (Yes I know this is a long process but that is what the contract we have with OCIO says we must do.)

- 16 SSD - We will add these to the SCO mentioned above then they can be iStored.

- NetApp physicals - Pricing has been complete and we can move to SCO for these

**Comment by jefkel** [comment]
Thanks @Joe-Craig , Do we have an ETA for having the SCO's above sent in for approval?

**Comment by tosazuwa** [comment]
SSD drives are priced and waiting to go onto an SCO. This SCO will also contain the 15 KPROD physical servers, which we are waiting on pricing. Murray to provide a quote for the 15 servers by EOD Wednesday Jan 29th

@Joe-Craig , i need a hard date for when the SCO will be sent in for approval. I know there are 2 meetings this week to finalize numbers. Once you have all the needed information, when can we expect to have the SCO submitted?

**Comment by Joe-Craig** [comment]
After meeting with Todd this is not a priority for this fiscal. SCO is complete.

**Comment by StevenBarre** [comment]
David Ell has asked we remove the SAN Fiber Channel cards from the 15 servers. Getting new quotes from HPE now and then updating the SCO.

**Comment by Joe-Craig** [comment]
New quote for servers requested. Will re-price and re-issue the SCO to reflect the changes requested by David Ell.

**Comment by Joe-Craig** [comment]
Delivery of new NetApp physicals to datacenters by end of March 2020.

**Comment by Joe-Craig** [comment]
SCO was returned by province with edits. Going back to the province today (9th March 2020)

**Comment by Joe-Craig** [comment]
SCO now signed by Province and procurement process is started.

**Comment by jefkel** [comment]
Update on expected arrival: 
June 8th.
